src/main/java/transportservice/RunPlugin.java

```diff
@@ -70,18 +75,37 @@ private void startTransport() throws UnknownHostException {
             transport,
             threadPool,
             NOOP_TRANSPORT_INTERCEPTOR,
-            connectionManager,
+            boundAddress -> DiscoveryNode.createLocal(
+                Settings.builder().put("node.name", "node_extension").build(),
```

Comment:
Same here, set it globally/pass a variable. Lets keep it consistent across the SDK.

Reply:
Will push the change in https://github.com/owaiskazi19/IndependentPlugin/pull/10 after we merge this PR.